apps/opik-frontend/src/components/pages/TracesPage/TracesActionsPanel.tsx

```diff
@@ -53,7 +54,15 @@ const TracesActionsPanel: React.FunctionComponent<TracesActionsPanelProps> = ({
         // we need split by dot to parse usage into correct structure
         const keys = column.split(".");
         const key = last(keys) as string;
-        acc[key] = get(row, keys, "");
+        const keyPrefix = first(keys) as string;
+
+        if (keyPrefix === "feedback_scores") {
+          acc[key] = (row.feedback_scores?.find((f) => f.name === key) ?? {
```

Comment:
`(row.feedback_scores?.find((f) => f.name === key) ?? "-"` 
looks simpler, or as you had below 
`get(row.feedback_scores?.find((f) => f.name === key), "value", "-")`

Reply:
That is a good suggestion, I will improve it with one of my next PRs.